### PR TITLE
ms auth guide - revoke clarifications

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/34206

### CODE REVIEW

File: content-repo/extra-docs/articles/microsoft-auth-guide.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ There are 2 application authentication methods available:
 2.  Self-Deployed Application
 
 ## Cortex XSOAR Application
-In this method, you give consent to the Cortex XSOAR application to access your data.
+In this method, you give consent to the Cortex XSOAR application, which we created in the Azure Portal, to access your data.
```
Comment (Itay4) [inline content-repo/extra-docs/articles/microsoft-auth-guide.md:15]:
```suggestion
In this method, you give consent to the Cortex XSOAR application, which we created in the Azure Portal, to access your data.
```
```suggestion
In this method, you give consent to the Cortex XSOAR multi-tenant application, which is maintained by Cortex XSOAR, to access your data.
```

File: content-repo/extra-docs/articles/microsoft-auth-guide.md [hunk 2]
```diff
@@ -64,9 +64,13 @@ For example, for Microsoft Graph User, replace the ***SCOPE*** with `directory.a
 9. Enter your redirect URI in the ***Redirect URI*** parameter field.
 
 
-## Revoke Consent
+## Revoking Consent
+In case you want to revoke the consent given to a usage of an application(on behalf of a user or an admin consent), please follow the instructions below:
+
+1. In order to revoke consent to a Cortex XSOAR Microsoft application, refer to the [Microsoft documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-convert-app-to-be-multi-tenant#revoking-consent).
```
Comment (Itay4) [inline content-repo/extra-docs/articles/microsoft-auth-guide.md:70]: need to work on the text here, you are writing basically twice "to revoke consent..."

File: content-repo/extra-docs/articles/microsoft-auth-guide.md [hunk 3]
```diff
@@ -64,9 +64,13 @@ For example, for Microsoft Graph User, replace the ***SCOPE*** with `directory.a
...
+2. Click on Access Panel Applications - either for users [link](https://myapplications.microsoft.com/) or for admins [link](https://portal.azure.com/#blade/Microsoft_AAD_IAM/StartboardApplicationsMenuBlade/AllApps).
+3. Use the same ID, Token and Key to redo the oProxy flow.
```
Comment (Itay4) [inline content-repo/extra-docs/articles/microsoft-auth-guide.md:72]: not sure i understand this point

**Comment by kirbles19** [comment]
@bakatzir Done

File: content-repo/extra-docs/articles/microsoft-auth-guide.md [hunk 4]
```diff
@@ -64,24 +64,6 @@ For example, for Microsoft Graph User, replace the ***SCOPE*** with `directory.a
 9. Enter your redirect URI in the ***Redirect URI*** parameter field.
 
 
-## Revoke Consent
-
-In order to revoke consent to a Cortex XSOAR Microsoft application, refer to the [Microsoft documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-convert-app-to-be-multi-tenant#revoking-consent). 
-
-## Azure Integrations Parameters
```
Comment (Itay4) [inline content-repo/extra-docs/articles/microsoft-auth-guide.md:71]: please revert this change

File: content-repo/extra-docs/articles/microsoft-auth-guide.md [hunk 5]
```diff
@@ -12,7 +12,7 @@ There are 2 application authentication methods available:
 2.  Self-Deployed Application
 
 ## Cortex XSOAR Application
-In this method, you give consent to the Cortex XSOAR application to access your data.
+In this method, you grant consent for the Cortex XSOAR multi-tenant application to access your data. The application is maintained by Cortex XSOAR, to access your data.
```
Comment (Itay4) [inline content-repo/extra-docs/articles/microsoft-auth-guide.md:15]:
```suggestion
In this method, you grant consent for the Cortex XSOAR multi-tenant application to access your data. The application is maintained by Cortex XSOAR.
```

**Review by Itay4 — APPROVED** [review]
